### PR TITLE
Update repository to reflect connect-es rename

### PR DESCRIPTION
**Pull request by smaye81**
Numerous changes to the repo to reflect Connect-ES renaming.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Connect Web Integration
+# Connect-ES Integration
 
-Connect Web Integration is a repository of example projects using Connect-Web with various JS frameworks and tooling.
-It provides numerous examples for integrating Connect-Web into a project.
+Connect-ES Integration is a repository of example projects using Connect-ES with various JS frameworks and tooling.
```
Comment (timostamm) [inline README.md:3]:
```suggestion
Connect-ES Integration is a repository of example projects using Connect with various TypeScript web frameworks and tooling.
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+It provides numerous examples for integrating Connect-ES into a project.
```
Comment (timostamm) [inline README.md:4]:
```suggestion
It provides numerous examples for integrating Connect into a project.
```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
-All projects test Connect-Web against the Eliza demo running at https://demo.connect.build.  Each project illustrates an example
+All projects test Connect-ES against the Eliza demo running at https://demo.connect.build.  Each project illustrates an example
```
Comment (timostamm) [inline README.md:6]:
```suggestion
All projects test Connect against the Eliza demo running at https://demo.connect.build.  Each project illustrates an example
```

**Review by timostamm — COMMENTED** [review]
IMO it works out to just use "Connect". What do you think?

**Comment by smaye81** [comment]
> IMO it works out to just use "Connect". What do you think?

sgtm.  I committed your suggestions.

**Review by timostamm — APPROVED** [review]

**Pull request merged**